Assets/Scripts/MusicController.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class MusicController : MonoBehaviour
+{
+	private AudioSource audioSource;
```

Comment:
removed 'music' member, found it unnecessary since AudioSource owns its own AudioClip